ui/js/controllers/perf/alerts.js

```diff
@@ -12,10 +12,29 @@ perf.factory('PhBugs', [
                         return alert.status !== phAlertStatusMap.INVALID;
                     });
                     var template = response.data[0];
+                    function getAlertSummary(alertSummary) {
+                        var resultStr = "";
+                        var improved = _.filter(alertSummary.alerts, function(alert) {return !alert.is_regression && alert.visible;});
+                        var regressed = _.filter(alertSummary.alerts, function(alert) {return alert.is_regression && alert.visible;});
+                        if (improved.length > 0) {
+                            resultStr += "Summary of tests that improved\n" +
```

Comment:
Could you add two newlines after "Summary of tests that improved". Also add a colon after "improved" (i.e. make it "improved:")